pkg/results/types.go

```diff
@@ -55,6 +55,7 @@ type ScanSummary struct {
 	LowCount             int    `json:"low" yaml:"low" xml:"low,attr"`
 	MediumCount          int    `json:"medium" yaml:"medium" xml:"medium,attr"`
 	HighCount            int    `json:"high" yaml:"high" xml:"high,attr"`
+	TotalTime            int64  `json:"-" yaml:"-" xml:"-"`
```

Comment:
should this be left blank for encoding?

Reply:
This is done on purpose, it is not required for other output types.